app/models/site_config.rb

```diff
@@ -148,7 +150,7 @@ class SiteConfig < RailsSettings::Base
   field :feed_style, type: :string, default: "basic"
   # a non-public forem will redirect all unauthenticated pages to the registration page.
   # a public forem could have more fine-grained authentication (listings ar private etc.) in future
-  field :public, type: :boolean, default: 1
+  field :public, type: :boolean, default: 0
```

Comment:
Changed the default state here from public to private. This makes private forem the default but can obviously be changed at config. Existing forems will already have a value here and the default won't need to apply.